src/script/ismine.cpp

```diff
@@ -28,6 +28,14 @@ enum class IsMineSigVersion
     WITNESS_V0 = 2  //! P2WSH witness script execution
 };
 
+enum class IsMineResult
```

Comment:
commit: Make handling of invalid in IsMine more uniform

If values were assigned to each (NO = 0, WATCH_ONLY = 1, SPENDABLE = 2, INVALID = 3), `Update` could be implemented as `if (update > val) val = update;`.

Reply:
Good idea, done.